File: tugas vfx 1-5/tugas1/tugas1.js
let jumlah_buku = 3;
let harga_buku = 45000;
let jumlah_pensil = 2;
let harga_pensil = 7500;
let jumlah_tas = 1;
let harga_tas = 120000;

let total_sebelum_diskon = (jumlah_buku * harga_buku) + (jumlah_pensil * harga_pensil) + (jumlah_tas * harga_tas);
let diskon = total_sebelum_diskon * 0.1;
let total_setelah_diskon = total_sebelum_diskon - diskon;
console.log("Total harga sebelum diskon adalah: " + total_sebelum_diskon);
console.log("Total harga setelah diskon adalah: " + total_setelah_diskon);

File: tugas vfx 1-5/tugas1/tugas1.cpp
#include <iostream>
using namespace std;

int main() {
    int jumlahb = 3;
    double hargab = 45000;
    int jumlahp = 2;
    double hargap = 7500;
    int jumlaht = 1;
    double hargat = 120000;
    double diskon = 0.10;

    double total = (jumlahb * hargab) + (jumlahp * hargap) + (jumlaht * hargat);

    std::cout << "Total harga sebelum diskon adalah: " << total << std::endl;

    double total_diskon = (total * diskon);

    cout << "Total harga setelah diskon adalah: " << (total - total_diskon) << std::endl;

    return 0;
}